default-filters.ts

```diff
@@ -155,66 +159,135 @@ export const defaultFilters: Record<string, Filter> = {
       vals: ['desc'],
       except: [],
     },
+    namespaces: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
   },
   'Complete: SCL Only (no Privates)': {
-    ...BASE_FILTER,
-    description: 'Compare the SCL namespace but ignore Private elements',
+    description:
+      'Compare the SCL namespace (but not Private elements or other namespaced items)',
     ourSelector: '',
     theirSelector: '',
     selectors: {
       inclusive: false,
       vals: ['Private'],
       except: [],
     },
+    attributes: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
     namespaces: {
       inclusive: true,
       vals: ['http://www.iec.ch/61850/2003/SCL'],
       except: [],
     },
   },
   Header: {
-    ...BASE_FILTER,
     description: 'Compare versioning and history items',
     ourSelector: 'Header',
     theirSelector: 'Header',
+    selectors: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
+    attributes: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
+    namespaces: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
   },
   Substation: {
-    ...BASE_FILTER,
     description: 'Compare single line diagram and specification functionality',
     ourSelector: 'Substation',
     theirSelector: 'Substation',
+    selectors: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
+    attributes: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
+    namespaces: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
   },
   'Communication: Complete': {
-    ...BASE_FILTER,
     description: '',
     ourSelector: 'Communication',
     theirSelector: 'Communication',
+    selectors: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
+    attributes: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
+    namespaces: {
+      inclusive: false,
+      vals: [],
+      except: [],
+    },
   },
   'Communication: IP Addresses and MMS Session': {
-    ...BASE_FILTER,
     description: 'Look at IP addresses',
```

Comment:
Sorry can we improve this a little?
```suggestion
    description: 'Compare IP address, subnet and gateway and MMS session parameters',
```